Community/Tdarr_Plugin_goof1_URL_Plex_Refresh.js

```diff
@@ -0,0 +1,221 @@
+
+const loadDefaultValues = require('../methods/loadDefaultValues');
+
+const details = () => ({
+  id: 'Tdarr_Plugin_goof1_URL_Plex_Refresh',
+  Stage: 'Post-processing',
+  Name: 'Refresh Plex via URL',
+  Type: 'Video',
+  Operation: 'Transcode',
+  Description: `Refreshes folder containing the current file in Plex so changes are picked up properly.`,
+  Version: '1.0',
+  Tags: '3rd party,post-processing,configurable',
+
+  Inputs: [
+    {
+      name: 'Url_Protocol',
+      type: 'string',
+      defaultValue: 'http',
+      inputUI: {
```

Comment:
Use dropdown menu UI here

Reply:
can you tell me where the ui types are listed? I looked through the repo and didn't see what the options were so i stuck with what i found in other extensions.